fastlane/Fastfile

```diff
@@ -17,27 +17,28 @@ opt_out_usage
 
 default_platform(:ios)
 
+COCOAPODS_PUSH_WAIT_TIME = 600
+
 pods = [
-  "Amplify.podspec",
-  "AWSPluginsCore.podspec",
-  "CoreMLPredictionsPlugin.podspec",
-  "AWSPredictionsPlugin.podspec",
-  "AmplifyPlugins.podspec"
+  {spec: "Amplify.podspec", sleep: true},
+  {spec: "AWSPluginsCore.podspec", sleep: false},
```

Comment:
Should we sleep after AWSPluginsCore as well? AmplifyPlugins depends on AWSPluginsCore

Reply:
I think the `sleep` after CoreMLPredictionsPlugin should cover that (since these are synchronous)

Reply:
Yeah, what he said. :)